packages/sycamore-macro/src/view/codegen.rs

```diff
@@ -375,6 +375,169 @@ impl CodegenTemplate {
                 });
                 (None, true)
             }
+            AttributeType::Spread => {
```

Comment:
All the code in the `quote!` should be extracted into a method in some utility `mod` in `sycamore`. We probably don't want to emit all this every time we use a spread because that would dramatically bloat the macro output and potentially slow down compile-times.

Reply:
You're right, it would've made writing it a lot easier too. Oh well, working on moving it and cleaning it up

Reply:
Done